### ISSUE TITLE
About The Training Loss

### ISSUE THREAD

**Issue by Ourano** [Sierkinhane/crnn_chinese_characters_rec#149]
At the beginning of the training, the loss value is very low, using the lmdb data, how should I check the data?

**Comment by Sierkinhane** [comment]
you can read the lmbd data or visualize images and label after data loading.
